src/lib/protocols/http.c

```diff
@@ -215,8 +215,11 @@ static void ndpi_validate_http_content(struct ndpi_detection_module_struct *ndpi
 
       if(len >= 8 /* 4 chars for \r\n\r\n and at least 4 charts for content guess */) {
 	double_ret += 4;
+	len -= 4;
```

Comment:
Tried to fix a possible buffer overflow in a previous commit, but I forgot why I've added it.
The reason is the end of the HTTP header `\r\n\r\n`. ;)